azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -1734,7 +1734,7 @@ private void cleanExecutionLogs() {
 
     private static final long QUEUE_PROCESSOR_WAIT_IN_MS = 1000;
     private final int maxDispatchingErrors;
-    private final long activeExecutorRefreshWindowInMilisec;
+    private final long activeExecutorRefreshWindowInMillisec;
```

Comment:
Thanks for fixing typos. 
In the future, I would suggest separate PRs for independent changes.